app/simformat.hs

```diff
@@ -64,6 +66,10 @@ parseOperation =
       <> Opt.metavar "TARGET"
       <> Opt.help "operate on a file in-place instead of reading from stdin and writing to stdout"
     repo = pure Repo
+    editor = Opt.flag' Editor
+      $ Opt.long "editor"
```

Comment:
This would typically be unimportant but since there are other changes I'll mention it. There is one space after the $ here but the above lines have two.